tools/packager/src/main/java/org/renjin/packaging/NamespaceBuilder2.java

```diff
@@ -177,7 +177,7 @@ private void writePackageRds() throws IOException {
     }
     File packageRdsFile = new File(metaDir, "package.rds");
     try(RDataWriter writer = new RDataWriter(HeadlessWriteContext.INSTANCE, new FileOutputStream(packageRdsFile))) {
-      writer.serialize(metadata.build());
+      writer.serialize(Context.newTopLevelContext(), metadata.build());
```

Comment:
Note the use of the HeadlessWriteContext.INSTANCE above. The intention is that a context is not required for pure data.